strategy/doubleTopDoubleBottom.py

```diff
@@ -0,0 +1,174 @@
+from binance.client import Client
+import pandas as pd
+from pyti.directional_indicators import positive_directional_index
+from pyti.directional_indicators import negative_directional_index
+from pyti.directional_indicators import average_directional_index
+import numpy as np
+from pyti.on_balance_volume import on_balance_volume
+from res import id as id, values as values, constants as constants
+import handler.database as db
+
+import plotly as py
+import plotly.graph_objs as go
+from scipy.signal import argrelextrema
+import talib
+
+#api_key = "YHuXjmYFOfG6RscRw8Z9sqTRy78FFE0sOOvCQsXIUpfxP1jx7MuBK4y8Bo42MUut"
+#secret_key = "w6Ipx3SDp8qorLaTE5xjSJhTcQc6CEkgOwRgwYl7JGi1SCsyrgQxG5zCa9ugnMPs"
+#moving_window = 45
+#window_size = 14
+
+#client = Client(api_key, secret_key)
+
+def get_local_min_max(closingPrices):
+    local_maxima = []
+    local_minima = []
+    indices_minima = []
+    indices_maxima = []
+    trend = 'neutral'
+    for i in range(len(closingPrices)-1):
+        #print(i)
+        if closingPrices[i+1] > closingPrices[i]:
+            #print("1")
+            if trend == 'decreasing':
+                #print(closingPrices[i])
+                local_minima.append(closingPrices[i])
+                indices_minima.append(i)
+                trend = 'increasing'
+            else:
+                #print("tp")
+                trend = 'increasing'
+        elif closingPrices[i+1] < closingPrices[i]:
+            #print("2")
+            if trend == 'increasing':
+                #print(closingPrices[i])
+                local_maxima.append(closingPrices[i])
+                indices_maxima.append(i)
+                trend = 'decreasing'
+            else:
+                #print("tp2")
+                trend = 'decreasing'
+        elif trend == 'neutral':
+            #print("3")
+            continue
+    print("Hi")
+    return local_minima, local_maxima, indices_minima,indices_maxima
+
+
+def check_double_top(price_action, indices_maxima, obv):
+
+    isDoubleTop = False
+
+    for i in range(0, len(indices_maxima) - 1):
+        highVal = price_action['High'].iloc[indices_maxima[i]]
```

Comment:
use id instead of 'Hight'. Also, High will not match with 'high' - hence the use of id to prevent such bugs